test_manage_arkime/core/test_capacity_planning.py

```diff
@@ -181,4 +181,41 @@ def test_WHEN_get_os_domain_plan_called_THEN_as_expected():
     )
     assert expected_value == actual_value
 
+def test_WHEN_cidr_created_THEN_as_expected():
```

Comment:
depend on answer above, probably should check that /29 etc aren't allowed since they are too small